e2e/cypress/e2e/spec.cy.js

```diff
@@ -227,4 +230,116 @@ describe("journey test", () => {
       });
     });
   });
+
+  // Testing Recurring events
+
+  describe("recurring events", () => {
+    it.only("displays monthly recurring events", () => {
+      cy.visit("http://localhost:3000");
+      cy.intercept({
+        method: "POST",
+        url: "/entries",
+        hostname: "localhost",
+      }).as("createEntry");
+
+      cy.contains("Morning run").should("not.exist");
+      cy.get(`[data-date="2022-12-22"]`).click();
+      cy.contains("label", "Title").click().type("Morning run");
+      cy.contains("label", "Description").click().type("Under the big blue");
+      cy.contains("label", "Start Date").click().type("2022-12-22");
+      cy.contains("label", "End Date").click().type("2022-12-22");
+
+      cy.contains("label", "Recurring").click();
+      cy.contains("label", "Recurrence End").click().type("2023-01-22");
```

Comment:
This event only goes for a single month. Is that intentional?

Reply:
If so I'd suggest changing the recurrence end to be a little bit later than when the recurrence happens, so maybe 1/25 instead of 1/22

Reply:
Yes it's intentional,  I wanted to check that the recurring event would display monthly recurring events while testing both the December month view and the January view. I will expand the Recurrence end date so it's more clear about with the expectation is.